requirements.txt

```diff
@@ -4,5 +4,8 @@ click
 wheel
 rlp>=0.4.4
 devp2p>=0.8.0
-git+https://github.com/ethereum/pyethereum.git@state_revamp
+git+https://github.com/ethereum/pyethereum.git@cba5954a43a2db4df7504d9ced70c63530247e77
 web3>=3.8.0
+pytest==3.4.0
+git+https://github.com/ethereum/vyper.git@a154d579062ae67cc3d79c942a30a384bcc1b24e
+ethereum-serpent==2.0.4
```

Comment:
Is serpent still required? We should remove the dependency if so. David Knott removed it in pyethereum somewhere around here: https://github.com/karlfloersch/pyethereum/blob/develop/ethereum/hybrid_casper/casper_initiating_transactions.py#L16

Reply:
I just removed the dependency using @changwu-tw's fix from PR #42.
#42 can be closed because the fix in that PR is to a file that is removed in this PR.